node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/task/util/env/package-info.java

```diff
@@ -0,0 +1,8 @@
+// Copyright 2019 Oath Inc. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+/**
+ * @author bjorncs
+ */
+@ExportPackage
+package com.yahoo.vespa.hosted.node.admin.task.util.env;
```

Comment:
What other files do you expect in this package? For now 1 file in util has been traded with 1 directory in util, and 2 files in util/env.

Reply:
Currently no other files. I am fine with moving it to the parent package.

Reply:
Done